src/runner/fullJSRunner.ts

```diff
@@ -38,7 +33,9 @@ function parseFullJS(code: string, context: Context): es.Program | undefined {
         start: { line: loc.line, column: loc.column },
         end: { line: loc.line, column: loc.column + 1 }
       }
-      context.errors.push(new FatalSyntaxError(location, error.toString()))
+      context.errors.push(
+        new FatalSyntaxError(location, acornSyntaxErrorMessageRegex.exec(error.toString())![0])
```

Comment:
Did we face any problems when not using the regex? If so I think we can use this implementation for error handling in `parser.ts` as well